pkgs/applications/graphics/drawing/default.nix

```
@@ -0,0 +1,72 @@
+{ stdenv
+, fetchFromGitHub
+, desktop-file-utils
+, file
+, gobject-introspection
+, pkgconfig
+, appstream-glib
+, meson
+, ninja
+, python3
+, gtk3
+, wrapGAppsHook
+, gettext
+, glib
+, python3Packages
+}:
+
+stdenv.mkDerivation rec {
+  pname = "drawing";
+  version = "0.4.10";
+
```

```suggestion


  format = "other";
```

this makes the python deriver use ninja and meson normally

---

thanks for sharing docs and lollypop's url !